Gemfile

```diff
@@ -35,6 +35,10 @@ group :test do
   gem 'govuk-content-schema-test-helpers', '~> 1.1.0'
 end
 
+group :test do
+  gem 'simplecov'
```

Comment:
We'll want simplecov-rcov as well so that we can get Jenkins to graph the output etc.